blocks/technology-table/technology-table.js

```diff
@@ -0,0 +1,70 @@
+function buildTechnologyTable(block) {
+  const firstDivElements = block.querySelectorAll('.technology-table > div');
+  firstDivElements.forEach((element, i) => {
+    const secondLevelDiv = element.querySelectorAll('div > div');
```

Comment:
can probably use `element.children` instead of doing a query per row

Reply:
Not really as the forEach very practically provides a the iteration number  (j in this case) that allows to dynamically add a number to the classes that we are adding to the table:

        subelement.classList.add(`techno-table-title-${j}`);
      } else if (i === 1) {
        subelement.classList.add(`techno-table-image-${j}`);
      } else if (i === 2) {
        subelement.classList.add(`techno-table-description-${j}`);
      }